build/stackedArea.js

```diff
@@ -0,0 +1,74 @@
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
this build file should be checked in? Is it a file that users of `britecharts` will use? Or is it the bundle for a public website?

Reply:
I still need to figure this out.
I think we would probably keep this whole build folder unchecked and add a 'final' step on a '/dist' folder.